components/elm/src/biogeochem/CH4Mod.F90

```diff
@@ -2563,8 +2563,7 @@ subroutine ch4_aere (bounds, &
                   is_vegetated = .false.
                end if
 
-               if (veg_pp%itype(p) == nc3_arctic_grass .or. crop(veg_pp%itype(p)) == 1 .or. &
-                    veg_pp%itype(p) == nc3_nonarctic_grass .or. veg_pp%itype(p) == nc4_grass) then
+               if (graminoid(veg_pp%itype(p)) == 1 .or. crop(veg_pp%itype(p)) == 1) then
```

Comment:
same as comments related to comparisons with woody, can we compare this to a named constant instead of 1?  Even if this is a integer acting as a logical, or a true logical, its better to compare to a named consant, like int_true or int_false.